Userland/Services/FileSystemAccessServer/Forward.h

```diff
@@ -0,0 +1,13 @@
+/*
```

Comment:
I'm not sure this file is needed, since it's only included by `ClientConnection` itself.